plugins/copr.py

```diff
@@ -179,6 +170,16 @@ def configure(self):
         else:
             copr_hub = self.opts.hub
 
+        # Check distribution and releasever in all config files
+        if config_files:
+
+            if len(config_files) > 1:
+                self._get_copr_chroot(config_files[1])
```

Comment:
Doesn't this mean that if there are more than one drop-in file, only the first one will be read?

Reply:
Yeah. Since I think there would only be 1 file with the branding anyway.

Reply:
Actually, I just realized that this could go very wrong. will fix